actions/access_keys/create.go

```diff
@@ -44,3 +44,19 @@ func (a *Action) create(c *gin.Context) {
 	contract := mappings.MapToAccessKeyContract(accessKey)
 	c.JSON(http.StatusCreated, contract)
 }
+
+// create will make a new model using the services defined in the action object
+// Create access key godoc
+// @Summary		Create access key
+// @Description	Create access key
+// @Tags		Access-key
+// @Produce		json
+// @Param		CreateAccessKey body CreateAccessKey true " "
+// @Success		201	{object} models.AccessKey "Created AccessKey"
+// @Failure		400	"Bad request - Error while parsing CreateAccessKey from request body"
+// @Failure 	500	"Internal server error - Error while creating new access key"
+// @Router		/v1/users/current/keys [post]
+// @Security	x-auth-xpub
```

Comment:
I'm would use single method, adjust summary and description and add 2 swagger routes: the new one, and the old one marked as @DeprecatedRouter

Reply:
done